tests/acceptance/features/apiCollaboration/checkFileInfo.feature

```diff
@@ -599,3 +599,620 @@ Feature: check file info with different wopi apps
       | Collabora     |
       | FakeOffice    |
       | OnlyOffice    |
+
+
+  Scenario Outline: try to get file info of deleted file with office suites
+    Given user "Alice" has uploaded file with content "hello world" to "/textfile0.txt"
+    And user "Alice" has sent POST request on app endpoint:
+      | resource        | textfile0.txt   |
+      | space           | Personal        |
+      | suites          | <office-suites> |
+    And user "Alice" has deleted file "/textfile0.txt"
+    When user "Alice" tries to get the file information of file using wopi endpoint
+    Then the HTTP status code should be "500"
```

Comment:
We can check the JSON response if it returns any response.

Reply:
looks like the 500 will be fixed soon

Reply:
> We can check the JSON response if it returns any response.

No JSON response was found so only http status code was checked. Same for 404 status code